osu.Game/Screens/Edit/Editor.cs

```diff
@@ -1129,6 +1133,14 @@ private EditorMenuItem createExportMenu()
             return new EditorMenuItem(CommonStrings.Export) { Items = exportItems };
         }
 
+        private void editExternally()
+        {
+            Save();
```

Comment:
Wouldn't it be wise to show a popup alerting the user that the beatmap will be saved in order to do this? Such as in https://github.com/ppy/osu/blob/efbbfab59de5c4b18279e7b74b8f1670c66a187f/osu.Game/Screens/Edit/Editor.cs#L482-L491

I think this would be a good idea in case the user forgot they made changes and want to revert them before the game commits them and opens the files for editing.